src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -330,13 +375,11 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
         // it expects to have the same data type for weights and zero points (apply it only for u8 data type, since other compression
         // types are not supported by oneDNN)
         manager.register_pass<ov::pass::MarkDequantizationSubgraph>(supported_woq_types, !device_info.supports_immad);
-
-        // Need to check if transformations work correctly for mixed models with both compression and quantization at the same time.
-        if (!is_model_quantized) {
-            pass_config->set_callback<ov::pass::MarkDequantizationSubgraph>([&](const std::shared_ptr<const ov::Node> node) {
-                return !is_decompression_multiply(node);
-            });
-        }
+        GPU_DEBUG_TRACE << "is_model_quantized: " << is_model_quantized << std::endl;
+        pass_config->set_callback<ov::pass::MarkDequantizationSubgraph>([&](const std::shared_ptr<const ov::Node> node) {
+            GPU_DEBUG_TRACE_DETAIL << node->get_friendly_name() << ": " << !is_decompression_multiply(node, device_info.supports_immad) << std::endl;
```

Comment:
I suggest removing these traces. They don't look useful for the future work, only for this particular patch.

Reply:
Applied